src/main/resources/application.yaml

```diff
@@ -1,2 +1,7 @@
 server:
   port: 8000
+clientregistry:
+  XAuthToken: c3f88ccc-2374-4a6d-bdba-63403e454979
```

Comment:
these should be all ${PROPERTY_NAME} names .. and not hardcoded. You may keep a "application-dev.yaml" file for such values.

Reply:
added application-dev.yaml file and removed hard coded properties